rsciio/pantarhei/_api.py

```diff
@@ -349,9 +349,13 @@ def _metadata_converter_in(meta_data, axes, filename):
         if "mm" in aperture:
             aperture = aperture.split("mm")[0]
             aperture = aperture.rstrip()
-        mapped.set_item(
-            "Acquisition_instrument.TEM.Detector.EELS.aperture_size", float(aperture)
-        )
+            mapped.set_item(
+                "Acquisition_instrument.TEM.Detector.EELS.aperture_size", float(aperture)
+            )
+        else:
+            mapped.set_item(
```

Comment:
It would be good to add a test to cover this line, for example, write a test which create a file with such aperture and read it again.
If you are not familiar with writing tests, you can have a look at https://hyperspy.readthedocs.io/en/latest/dev_guide/testing.html. The test should go in https://github.com/hyperspy/rosettasciio/blob/main/rsciio/tests/test_pantarhei.py. Of course, you can use other tests as source of inspiration!